scripts/main.ps1

```diff
@@ -81,6 +81,19 @@ try {
     Write-Host "Unable to load Windows.Markup.XamlReader. Double-check syntax and ensure .net is installed."
 }
 
+
+# Load the configuration files
+#Invoke-WPFUIElements -configVariable $sync.configs.nav -targetGridName "WPFMainGrid"
+$sync.form.Dispatcher.Invoke([action] {
+    Invoke-WPFUIElements -configVariable $sync.configs.applications -targetGridName "appspanel" -columncount 5
+})
+$sync.form.Dispatcher.Invoke([action] {
+    Invoke-WPFUIElements -configVariable $sync.configs.tweaks -targetGridName "tweakspanel" -columncount 2
+})
+$sync.form.Dispatcher.Invoke([action] {
+    Invoke-WPFUIElements -configVariable $sync.configs.feature -targetGridName "featurespanel" -columncount 2
+})
+
```

Comment:
@MyDrift-user Because this's better off with actual parallel processing (to speedup the generation of UI Elements), and current use of `$sync.form.Dispatcher` doesn't actually do this parallelism... it's a good idea to leave a **TODO** so your future self or other contributor can try and fix this, either through simple patches.. or by some code refactoring to achieve a similar goal (making WinUtil startup feel more snappier/responsive)

Reply:
Noticed that you left a Note in `Invoke-WPFUIElements` ([here's the note](https://github.com/ChrisTitusTech/winutil/pull/2465/files#diff-6432879fb79af1049a6c704f31a332fc11ff727b13434d79bb63d811d8e1b01aR13-R14)), my bad @MyDrift-user 😅